oracle/datadog_checks/oracle/oracle.py

```diff
@@ -101,7 +101,11 @@ def _fix_custom_queries(self):
                 if prefix.startswith(self.__NAMESPACE__ + '.'):
                     prefix = prefix[len(self.__NAMESPACE__) + 1 :]
                 for column in query.get('columns', []):
-                    if column.get('type') != 'tag':
+                    # if the column is supposed to be skipped
+                    if not column.get('type') and not column.get('name'):
```

Comment:
maybe this is 'or' condition?

Reply:
I can change this to `not (column.get('type') or not column.get('name'))` instead if that can be clearer.